src/components/SessionListItem/SessionListItem.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../styles/abstracts/_mixins.scss";
+@import "../../styles/abstracts/_variables.scss";
+
+.wrapper{
+    @include component;
+  
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
```

Comment:
This margin makes the list item not align with the header. Can be removed, and rather use `gap: 10px` in the SessionList.module.scss file.